src/js/components/message.jsx

```diff
@@ -16,11 +19,14 @@ export class MessageComponent extends Component {
         accentColor: PropTypes.string,
         linkColor: PropTypes.string,
         firstInGroup: PropTypes.bool,
-        lastInGroup: PropTypes.bool
+        lastInGroup: PropTypes.bool,
+        sendStatus: PropTypes.string,
+        entity: PropTypes.object.isRequired
     };
 
     static defaultProps = {
-        actions: []
+        actions: [],
+        sendStatus: 'sent'
```

Comment:
can you make a `SEND_STATUS` constant somewhere and have the status string there?